src/components/Modal.tsx

```diff
@@ -0,0 +1,27 @@
+import "./Modal.scss";
+import Button from "./Button";
+import { ReactNode } from "react";
+
+interface ModalProps {
+  children?: ReactNode;
+  closeAction: () => void;
+}
+
+const Modal = ({ children, closeAction }: ModalProps) => {
+  return (
+    <div className="Gallery">
```

Comment:
The modal shouldn't have anything to do with the gallery (it should be reusable), so I think this outer `div` should be removed completely